Formula/conan.rb

```diff
@@ -11,7 +11,11 @@ class Conan < Formula
     sha256 "aae8d317e2e9add74d893d0183c3f93539f98b81f59f67bed1d1f2a9466198bd" => :mavericks
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
+  resource "backport_ipaddress" do
```

Comment:
Please leave an empty line between the `depends_on` and the `resource` line. Also, why did you drop the `if MacOS.version <= :snow_leopard` on the `depends_on :python` line?